src/main/java/com/boggle/BoggleAnnotations.java

```diff
@@ -0,0 +1,94 @@
+package com.boggle;
+
+import com.google.cloud.vision.v1.EntityAnnotation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BoggleAnnotations {
+    EntityAnnotation entityAnnotation;
+    // all these vertices are null: please change
+    Vertex topLeft;
+    Vertex topRight;
+    Vertex bottomLeft;
+    Vertex bottomRight;
+    int widthActual;
+    int heightActual;
+    int rowPixelIncrement;
+    int colPixelIncrement;
+    String description;
+
+    public static void main(String[] args) {
+    }
+
+    public BoggleAnnotations(EntityAnnotation newEntity, int width, int height, String description) {
+        this.entityAnnotation = newEntity;
+        widthActual = width;
+        heightActual = height;
+        rowPixelIncrement = widthActual / 5;
+        colPixelIncrement = heightActual / 5;
+        this.topLeft = new Vertex(entityAnnotation.getBoundingPoly().getVertices(0).getX(), entityAnnotation.getBoundingPoly().getVertices(0).getY());
+        this.topRight = new Vertex(entityAnnotation.getBoundingPoly().getVertices(1).getX(), entityAnnotation.getBoundingPoly().getVertices(1).getY());
+        this.bottomRight = new Vertex(entityAnnotation.getBoundingPoly().getVertices(2).getX(), entityAnnotation.getBoundingPoly().getVertices(2).getY());
+        this.bottomLeft = new Vertex(entityAnnotation.getBoundingPoly().getVertices(3).getX(), entityAnnotation.getBoundingPoly().getVertices(3).getY());
+        description.replaceAll("\\s+", "");
+        this.description = description;
+    }
+
+    public int getBottomLeftX() {
+        bottomLeft.x = entityAnnotation.getBoundingPoly().getVertices(3).getX();
```

Comment:
for all these getters why are you recalculating bottlmLeft.x and trying to reassign? why cant you use the private variable here and make this look something like `return this.bottomLeft.x`?